sdk/core/azure-core/src/main/java/com/azure/core/models/spatial/GeometryBoundingBox.java

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.models.spatial;
+
+import java.util.Objects;
+
+/**
+ * Represents a geometric bounding box.
+ */
+public final class GeometryBoundingBox {
+    private final double west;
+    private final double south;
+    private final double east;
+    private final double north;
+
+    private final Double minAltitude;
+    private final Double maxAltitude;
+
+    /**
+     * Constructs a bounding box.
+     *
+     * @param west West longitudinal boundary.
+     * @param south South latitudinal boundary.
+     * @param east East longitudinal boundary.
+     * @param north North latitudinal boundary.
+     */
+    public GeometryBoundingBox(double west, double south, double east, double north) {
```

Comment:
I was wondering if this could be named differently or to be more specific into the spatial concept that we would foresee it being used. Or do we want this same model to be more generic and allow including the concept of the `BoundingBox` we encountered in `FormRecognizer` [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json#L1315).


 Where a `BoundingBox` could be a `"Quadrangle bounding box, with coordinates specified relative to the top-left of the original image. The eight numbers represent the four points, clockwise from the top-left corner relative to the text orientation. For image, the (x, y) coordinates are measured in pixels. For PDF, the (x, y) coordinates are measured in inches.",`

Reply:
I believe these concepts are distinct from each other. The GeoJSON bounding box must be a rectangle as it simply lists to left, right, top, bottom bounding areas, with optional depth. Form Recognizer uses the four corners of the bounding area so it is a quadrangle which is a super set from a rectangle.

Reply:
I think it's fair to keep them as two separate models especially considering the fact that FR service is thinking of updating the `BoundingBox` to not be restricted to just four points and it can then grow differently.

But still think that `GeometryBoundaryBox` model name is too generic and can confuse users once we start including the other BoundingBox concepts. It doesn't suggest it is a GeoJson bounding box.  What do you think?

Reply:
That's a good thought.

@pakrym, @srnagar, @JonathanGiles, @KrzysztofCwalina, ideas here as this will be common across the languages

Reply:
I'm ok with how it is named now, but I am open to suggestions for other names too.

Reply:
To me the prefix seems reasonable to distinguish.